src/BreakfastSerializer/Property/TypeHandler/DateTimeHandler.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace BDBStudios\BreakfastSerializer\Property\TypeHandler;
+
+trait DateTimeHandler
```

Comment:
Yeah, is why it says do not merge, not even ready for review :D

Have to integrate it into the serializer, and make it bypassable via config
then it is review ready


On Tue, Jul 28, 2015 at 1:20 PM, Eddie Abou-Jaoude <notifications@github.com
> wrote:

> In src/BreakfastSerializer/Property/TypeHandler/DateTimeHandler.php
> <https://github.com/brettminnie/breakfast-serializer/pull/34#discussion_r35640263>
> :
>
> > @@ -0,0 +1,43 @@
> > +<?php
> > +
> > +namespace BDBStudios\BreakfastSerializer\Property\TypeHandler;
> > +
> > +trait DateTimeHandler
>
> Docbloc
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brettminnie/breakfast-serializer/pull/34/files#r35640263>
> .
>